docs/team/johndoe.md

```diff
@@ -1,46 +0,0 @@
----
```

Comment:
I don't think we need to delete this file as we can still refer to it in the future when we update our PPPs.